packages/applications-service-api/src/services/document.service.js

```diff
@@ -47,11 +47,56 @@ const getDocuments = async (caseRef, pageNo, searchTerm) => {
   return documents;
 };
 
-const getOrderedDocuments = async (caseRef, pageNo) => {
+const getOrderedDocuments = async (caseRef, pageNo, searchTerm, stage, type) => {
   const { itemsPerPage: limit } = config;
   const offset = (pageNo - 1) * limit;
 
-  const where = { case_reference: caseRef };
+  let where = { case_reference: caseRef };
```

Comment:
We need to make sure the base where condition includes `Stage: { [Op.gt]: 0 }`

See https://github.com/Planning-Inspectorate/applications-service/pull/237/files#diff-b4447c6a47b8ab588745de2cd533acd987452209f382cc2fe2315fde320f53f0R54

Reply:
Can we ever have a Stage value less than 0 ?

Reply:
Not by intention from the UI but we need to make sure we always filter out documents with Stage 0 since they are the documents attached to registration comments!